### PR TITLE
Test Language and App containers only on 15.5 by default

### PR DESCRIPTION
**Pull request by dirkmueller**
Introduce a new DEFAULT setting that is globally applied for testing if no more specific setting is being provided. Skip testing of application and language containers on 15.4 now that we have rolled over to 15.5

### CODE REVIEW

File: bci_tester/data.py [hunk 1]
```diff
@@ -31,19 +31,33 @@
...
+# Test Language and Application containers by default for these versions
+_DEFAULT_NONBASE_OS_VERSIONS = ("15.5",)
+
+assert sorted(ALLOWED_BASE_OS_VERSIONS) == list(
+    ALLOWED_BASE_OS_VERSIONS
+), f"list ALLOWED_OS_VERSIONS must be sorted, but got {ALLOWED_BASE_OS_VERSIONS}"
+
+assert sorted(ALLOWED_NONBASE_OS_VERSIONS) == list(
+    ALLOWED_NONBASE_OS_VERSIONS
+), f"list ALLOWED_NONOS_VERSIONS must be sorted, but got {ALLOWED_NONBASE_OS_VERSIONS}"
```
Comment (dcermak) [inline bci_tester/data.py:49]: Why did you add this part? It's not wrong, but it's not user modifiable, so why?

Comment (dirkmueller) [reply bci_tester/data.py:49]: No particular reason, just to give hints to my future self on when the next times those settings/variables/constants are modified which rules they need to comply with.
